GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -711,12 +722,12 @@ protected override void DisposeCustomResources()
 
         private sealed class ItemTpComparer : IComparer<string>
         {
-            private readonly IList<string> _otherList;
+            private readonly IDictionary<string, int> _orderDict;
```

Comment:
Field/param in this type should match outer type IMO.

Reply:
I don't know what you mean. Could you explain it?

Reply:
Sure, I mean in this case the field's called `_orderDict` and it's called `_sortedRefs` in the outer scope. They're the same object and it'd be clearer if they had the same name.

Reply:
Thank you, now it is clear.